app/controllers/school_search_controller.rb

```diff
@@ -21,7 +21,7 @@ def search_schools
     rescue ArgumentError => e
       raise unless e.message == School::SEARCH_NOT_ENOUGH_CHARACTERS_ERROR
 
-      errors.push("'query' parameter must have a minimum of four characters")
+      errors.push("'query' parameter must have a minimum of three characters")
```

Comment:
Would it make sense to use the `SEARCH_NOT_ENOUGH_CHARACTERS_ERROR` here?

Reply:
I've reworded `SEARCH_NOT_ENOUGH_CHARACTERS_ERROR` to make more sense outside the model and reused it here 👍

Reply:
Kewl 😃